index.js

```diff
@@ -110,3 +125,10 @@ function getWordsUntilLength(words, maxLength) {
 
   return wordsOutput;
 }
+
+function croppWordToGivenSize(word, outputLength) {
+  return word
+    .split('')
+    .slice(0, outputLength)
```

Comment:
By the way, `.slice()` works on strings as well, so no need to `split` & `join`

![image](https://user-images.githubusercontent.com/3817380/43402460-765ecd14-941b-11e8-87e1-1333dc52d2b0.png)